### PR TITLE
tier-0 test suite for build qualification

### PR DESCRIPTION
**Pull request by rkhambad**
Signed-off-by: Rajendra Khambadkar <rkhambad@rkhambad.remote.csb>

# Description

Tier-0 test suite for build qualification.

### CODE REVIEW

File: conf/quincy/cephadm/tier-0.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# System Under Test environment configuration for Tier-o test suites for build qualification.
```
Comment (Ameenasuhani) [inline conf/quincy/cephadm/tier-0.yaml:1]: Typo Tier-0

**Comment by Ameenasuhani** [comment]
@rkhambad Can you attach logs of successful run?

File: conf/quincy/cephadm/tier-0.yaml (hunk 1 above)
Comment (rkhambad) [reply conf/quincy/cephadm/tier-0.yaml]: Done.

File: conf/quincy/cephadm/tier-0.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# System Under Test environment configuration for Tier-0 test suites for build qualification.
```
Comment (psathyan) [inline conf/quincy/cephadm/tier-0.yaml:1]: IMO, we can reuse `minimal.yaml` in `conf` folder.

**Comment by rkhambad** [comment]
@Ameenasuhani  right now we don't have the downstream build to run the test suite.
Is there any other way u know to execute the test suite?

**Comment by Ameenasuhani** [comment]
@rkhambad We can test it on 5.2 itself for now to make sure we don’t hit any issues, later we can test it on 6.0 builds in  pipeline or skip for now

**Comment by rkhambad** [comment]
For 5.2 we are already running it in pipeline as tier-0.
Will skip logs for now.

**Comment by psathyan** [comment]
IMO, we would like to keep the pipeline green and we can wait till it is tested.

File: conf/quincy/cephadm/tier-0.yaml (hunk 2 above)
Comment (psathyan) [reply conf/quincy/cephadm/tier-0.yaml]: @rkhambad could you let me why we are creating a separate yaml file?

**Comment by Ameenasuhani** [comment]
@psathyan I think the plan was to test 6.0 using pipeline at first place itself @veera-raghava-reddy So should we wait until the builds arrive?

**Comment by rkhambad** [comment]
1) Tier-0 Test Result for 6.0 build are located at:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-CYJTM6/ 

2) Conf file used from location: 
conf/3-node-cluster-with-1-client.yaml

3) RBD IO test case has issue with 6.0 build and failed with below error:
**2022-07-25 07:07:18,275 - ERROR - cephci.cephfs_basic_tests:110 - yum install -y xmlstarlet rbd-nbd qemu-img cryptsetup --nogpgcheck Error:  Failed to set locale, defaulting to C.UTF-8
Error: Unable to find a match: rbd-nbd
 10.0.210.165**

 4) 6.0 build taken from below link: 
http://file.rdu.redhat.com/~kdreyer/osbs/RHCEPH-6.0-RHEL-9.json

File: conf/quincy/cephadm/tier-0.yaml (hunk 2 above)
Comment (psathyan) [reply conf/quincy/cephadm/tier-0.yaml]: For now... we can leave it. If we go to the root folder, custom test execution would break.

**Comment by psathyan** [comment]
> 1. Tier-0 Test Result for 6.0 build are located at:
>    http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-CYJTM6/
> 2. Conf file used from location:
>    conf/3-node-cluster-with-1-client.yaml
> 3. RBD IO test case has issue with 6.0 build and failed with below error:
>    **2022-07-25 07:07:18,275 - ERROR - cephci.cephfs_basic_tests:110 - yum install -y xmlstarlet rbd-nbd qemu-img cryptsetup --nogpgcheck Error:  Failed to set locale, defaulting to C.UTF-8
>    Error: Unable to find a match: rbd-nbd
>    10.0.210.165**
> 4. 6.0 build taken from below link:
>    http://file.rdu.redhat.com/~kdreyer/osbs/RHCEPH-6.0-RHEL-9.json

Please raise a BZ for `rbd-nbd` failure.

**Review by psathyan — APPROVED** [review]

**Pull request merged**